src/compression.h

```diff
@@ -164,7 +164,7 @@ std::unique_ptr<char[]> uncompress(const zim::Reader* reader, zim::offset_t star
   // Use a compressor to compress the data.
   // As we don't know the result size, neither the compressed size,
   // we have to do chunk by chunk until decompressor is happy.
-  // Let's assume it will be something like the minChunkSize used at creation
+  // Let's assume it will be something like the default clusterSize used at creation
   Uncompressor<INFO> runner(1024*1024);
```

Comment:
If this value is supposed to be the same as the default cluster size, then we better use a shared constant in `src/constants.h`.